### PR TITLE
Add release notes for 7.9.1

### PR DESCRIPTION
**Pull request by Mpdreamz**
## Preview

https://elasticsearch_61861.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/7.9/release-notes-7.9.1.html

### CODE REVIEW

File: docs/reference/release-notes/7.9.asciidoc [hunk 1]
```diff
@@ -1,9 +1,110 @@
...
+* Fix wrong pipeline name in debug log {es-pull}58817[#58817] (issue: {es-issue}58478[#58478])
+* Fix wrong result when executing bulk requests with and without pipeline {es-pull}60818[#60818] (issue: {es-issue}60437[#60437])
+* Update regex file for es user agent node processor {es-pull}59697[#59697] (issue: {es-issue}59694[#59694])
+
+IdentityProvider::
+* Only call listener once (SP template registration) {es-pull}60497[#60497] (issues: {es-issue}54285[#54285], {es-issue}54423[#54423])
+
+Machine Learning::
+* Always write prediction_[score|probability] for classification inference {es-pull}60335[#60335]
+* DFA Get Stats can return multiple responses if more than one error {es-pull}60900[#60900] (issue: {es-issue}60876[#60876])
```
Comment (lcawl) [inline docs/reference/release-notes/7.9.asciidoc:80]:
```suggestion
* Get data frame analytics stats API can return multiple responses if more than one error {es-pull}60900[#60900] (issue: {es-issue}60876[#60876])
```

File: docs/reference/release-notes/7.9.asciidoc [hunk 2]
```diff
@@ -1,9 +1,110 @@
...
+
 Also see <<breaking-changes-7.9,Breaking changes in 7.9>>.
 
-coming::[7.9.1]
+[[feature-7.9.1]]
+[float]
+=== New features
+
+Search::
+* QL: wildcard field type support {es-pull}58062[#58062] (issues: {es-issue}54184[#54184], {es-issue}58044[#58044])
```
Comment (jrodewig) [inline docs/reference/release-notes/7.9.asciidoc:13]:
```suggestion
* QL: Wildcard field type support {es-pull}58062[#58062] (issues: {es-issue}54184[#54184], {es-issue}58044[#58044])
```

File: docs/reference/release-notes/7.9.asciidoc [hunk 3]
```diff
@@ -1,9 +1,110 @@
...
+EQL::
+* Make endsWith function use a wildcard ES query wherever possible {es-pull}61160[#61160] (issue: {es-issue}61154[#61154])
+* Make stringContains function use a wildcard ES query wherever possible {es-pull}61189[#61189] (issue: {es-issue}58922[#58922])
+
+Features/Stats::
+* Change severity of negative stats messages from WARN to DEBUG {es-pull}60375[#60375]
+
+Search::
+* Fix handling of alias filter in SearchService#canMatch {es-pull}59368[#59368] (issue: {es-issue}59367[#59367])
+* QL: add filtering query dsl support to IndexResolver {es-pull}60514[#60514] (issue: {es-issue}57358[#57358])
```
Comment (jrodewig) [inline docs/reference/release-notes/7.9.asciidoc:33]:
```suggestion
* QL: Add filtering Query DSL support to IndexResolver {es-pull}60514[#60514] (issue: {es-issue}57358[#57358])
```

**Review by jrodewig — APPROVED** [review]
LGTM. Left some minor fixes.

File: docs/reference/release-notes/7.9.asciidoc [hunk 4]
```diff
@@ -1,9 +1,110 @@
...
+* Fix handling of final pipelines when destination is changed {es-pull}59522[#59522] (issue: {es-issue}57968[#57968])
+* Fix wrong pipeline name in debug log {es-pull}58817[#58817] (issue: {es-issue}58478[#58478])
+* Fix wrong result when executing bulk requests with and without pipeline {es-pull}60818[#60818] (issue: {es-issue}60437[#60437])
+* Update regex file for es user agent node processor {es-pull}59697[#59697] (issue: {es-issue}59694[#59694])
+
+IdentityProvider::
+* Only call listener once (SP template registration) {es-pull}60497[#60497] (issues: {es-issue}54285[#54285], {es-issue}54423[#54423])
+
+Machine Learning::
+* Always write prediction_[score|probability] for classification inference {es-pull}60335[#60335]
```
Comment (lcawl) [inline docs/reference/release-notes/7.9.asciidoc:79]:
```suggestion
* Always write prediction_probability and prediction_score for classification inference {es-pull}60335[#60335]
```

**Review by lcawl — APPROVED** [review]
Minor suggestions, otherwise LGTM

**Pull request merged**